### DISCUSSION TITLE
🔗 Customized icons & URLs for Floating Button widget (Resource links) (Merged with #28741)

### DISCUSSION THREAD

**Discussion by reedock** [elementor/elementor#28136] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

As of today the link used to trigger the opening of Messenger on Mobile doesn't work.
Bug reported here: https://github.com/elementor/elementor/issues/28103

If Meta, Viber, Instagram etc.. changes their URL tomorrow for triggering their own platforms, all the users of this widget will be affected. 

We’ll have to wait for someone to report it and then wait for an update from Elementor that could take days or weeks and that's millions of websites that will be affected..

Having the possibility to customized URLs will be able to give an instant fix to be spread to everyone while waiting for an official fix. 

### Describe the solution you'd like.

Add the possibility to customize our own URL & Icons for the Resource links would be better to allow multiple scenarios.

### Describe alternatives you've considered.

Icon widget with customized URL.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by toineenzo** [comment]
I think it's in the works, check https://github.com/elementor/elementor/pull/28020

**Comment by nicholaszein** [comment]
Hi there,

### 🔃 This Feature Request is being merged with #28741 which is on the same subject. 

if you still think this request should be considered for our roadmap, please cast your **⬆️ Upvote** on the first comment.

Don't worry, your comments, reactions, and participation in this thread will be accounted for when considering this Feature Request for our Roadmap.

Happy voting! 🙂
